.travis.yml

```diff
@@ -15,13 +15,14 @@ cache:
     - $HOME/gopath/pkg/mod
 
 before_install:
+  - nvm install 12.10.0
```

Comment:
Is there a reason you used this specific version? The latest is 12.11.0

Reply:
No real reason, we just need a newer version than the one that is installed by default because of yarn.